addons/account/demo/account_invoice_demo.yml

```diff
@@ -52,6 +52,27 @@
   !python {model: account.invoice, id: demo_invoice_3}:
     self.action_invoice_open()
 -
+   Create an invoice that will generate a followup
+-
```

Comment:
@csnauwaert I don't see how you can ensure there will be a followup for that invoice on the 1st day of the month